src/layout/msWorkspace/msWorkspace.js

```diff
@@ -43,9 +44,18 @@ var MsWorkspace = class MsWorkspace {
         }
 
         this.msWorkspaceActor = new MsWorkspaceActor(this);
-        const Layout = Me.tilingManager.getLayoutByKey(
-            initialState ? initialState.tilingLayout : 'maximized'
-        );
+        let defaultLayout = getSettings('layouts').get_string('defaultlayout');
+        let Layout;
+        if (defaultLayout !== '' && !initialState) {
+            Layout = Me.tilingManager.getLayoutByKey(
+                defaultLayout
+            );
+        } else {
+            Layout = Me.tilingManager.getLayoutByKey(
+                initialState ? initialState.tilingLayout : 'maximized'
+            );
+        }
```

Comment:
Since we have now a default value set as `'maximized'` we can simplify this as:
```
const Layout = Me.tilingManager.getLayoutByKey(
            initialState ? initialState.tilingLayout : defaultLayout
       );
```

Reply:
done